app/views/supermarkets/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h2><%= @supermarket.name %></h2>
+<br>
+<h4>Customers:</h4>
+<% @customers.each do |customer| %>
```

Comment:
Here we would probably want to make a method in our `supermarket` model that grabs unique customers and then calls that method on our `@supermarket`. See my comments in your controller.